ports/mtl/vcpkg.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "mtl",
+  "version-string": "1.0.0",
```

```suggestion
  "version": "1.0.0",
```
if it is 1.0.0 at all - I don't spot it on https://github.com/tonitaga/Matrix-Template-Library-CPP/tags.

---

From now on the version numbering will start again, I didn't have time to change it in my repository